`global.conf` initialization (#3211)
conan-io/docs: reference/config_files/global_conf.rst

global.conf
===========

The **global.conf** file is located in the Conan user home directory, e.g., *[CONAN_HOME]/global.conf*. If it does not

> already exist, a default one will automatically be created.

Reviewer: ```suggestion
already exist, a default one is automatically created.
```
